ludwig/schema/utils.py

```diff
@@ -984,7 +985,8 @@ def _serialize(self, value, attr, obj, **kwargs):
                 try:
                     if value is None and mfield_meta.allow_none:
                         return None
-                    mfield_meta.validate(value)
+                    if mfield_meta.validate:
+                        mfield_meta.validate(value)
```

Comment:
Consider adding a comment about why this might be significant.

Reply:
added